src/applications/vaos/new-appointment/redux/selectors.js

```diff
@@ -400,3 +400,9 @@ export function selectFacilitiesRadioWidget(state) {
 export function selectAppointmentSlotsStatus(state) {
   return getNewAppointment(state).appointmentSlotsStatus;
 }
+
+export function getSelectedDate(state) {
+  return getFormData(state).selectedDates?.length
+    ? getFormData(state).selectedDates[0]
+    : '';
+}
```

Comment:
Is '' a valid value for the CalendarWidget?

Reply:
No, the calendar widget wants an array, I am setting this to an empty string because the other return is also a string. On the page this gets put in an array. I am wondering now though if this should get put in a new file in `src/applications/vaos/appointment-referral/redux/selectors.js`